VEHICLE/SetCheckForEnoughRoomForPed.md

```diff
@@ -2,11 +2,11 @@
 ns: VEHICLE 
 ---
 
-## _0xEF9D388F8D377F44
+## SET_CHECK_FOR_ENOUGH_ROOM_FOR_PED
 
 ```c
 // 0xEF9D388F8D377F44 
-void _0xEF9D388F8D377F44(Vehicle vehicle, BOOL p1);
+void SET_CHECK_FOR_ENOUGH_ROOM_FOR_PED(Vehicle vehicle, BOOL p1);
```

Comment:
Second param is to check if it should check for enough room to fid the current ped.